### ISSUE TITLE
Summary export of concentrations in projects

### ISSUE THREAD

**Issue by jiho** [ecotaxa/ecotaxa_front#616]
The pseudo code to compute the concentration per sample is

```
GROUP BY sample, category
SUM individual_concentration
```

where `individual_concentration` is

```
individual_concentration = 1 / subsample_coef / total_water_volume
```

where:
- `total_water_volume` is the volume of water that was sampled, in m3
- `subsample_coef` is a number in [0,1] that represents the fraction of that total volume that was actually imaged

See #619 for how they are specified.

`subsample_coef` is a subsample (acquisition) property.

If the `total_water_volume` is specified at `sample` level (e.g. ZooScan, FlowCam), then this is computed per sample. 
If the `total_water_volume` is specified at `subsample` level (e.g. ISIIS, UVP), then this is computed per subsample. 

Then all the same considerations as for abundances (#615) apply regarding 0 and the shape of the output data.

**Comment by jiho** [comment]
As per another issue: re-use the code in 
https://github.com/ecotaxa/ecotaxa_back/blob/4fa0cbaaa1b3d7b35699111703f47dc042dea61f/py/API_operations/exports/EMODnet.py#L427

**Comment by grololo06** [comment]
⚠️ #601

**Comment by grololo06** [comment]
Whet is 'this' in:
>If the total_water_volume is specified at sample level (e.g. ZooScan, FlowCam), then this is computed per sample.
>If the total_water_volume is specified at subsample level (e.g. ISIIS, UVP), then this is computed per subsample.

Examples are needed as well.

**Comment by grololo06** [comment]
OK it's not really possible to re-use the DwC code which has a hard-coded aggregation level (sample), so we go for a rewrite.

**Comment by grololo06** [comment]
Is it OK?
    concentrations: Final = Term("Abundance of biological entity specified elsewhere per unit volume of the water body",
                                 "http://vocab.nerc.ac.uk/collection/P01/current/SDBIOL01/")

**Comment by grololo06** [comment]
API side implemented in v2.6.8.
